liblouis/lou_translateString.c

```diff
@@ -3432,11 +3439,13 @@ translateString(const TranslationTableHeader *table, int mode, int currentPass,
 	markEmphases(table, input, typebuf, wordBuffer, emphasisBuffer, haveEmphasis);
 
 	while (pos < input->length) { /* the main translation loop */
-		if ((pos >= compbrlStart) && (pos < compbrlEnd)) {
+		if (pos >= compbrlStart && pos < compbrlEnd) {
```

Comment:
I had to look this up in an c operator preference chart. And yes it is apparently so that `>=` is stronger than `&&`. However I personally like it much better if the code would not contain such traps where I have to look things up in a table. In other words in this case I'd prefer the brackets to make things clear for the reader.
But wait, the rest of the code is exactly like the code you fixed, so made the code look like the rest. So I approve the change :-)